From 092e14211aadf22dd50a3a7cf1c712b721cc395f Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Tue, 4 Sep 2018 18:07:54 +0800
Subject: [PATCH] arm64: dts: rockchip: add io-domain for rk1808

Add support for rk1808 io domain and pmu io domain.

Change-Id: I1057f9193ca12a8e14a5dfa0b121395169470b0f
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index d6c13ab61e8b..c5b89ee1fc02 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -123,6 +123,11 @@
 		reg = <0x0 0xfe000000 0x0 0x1000>;
 		#address-cells = <1>;
 		#size-cells = <1>;
+
+		io_domains: io-domains {
+			compatible = "rockchip,rk1808-io-voltage-domain";
+			status = "disabled";
+		};
 	};
 
 	usb2phy_grf: syscon@fe010000 {
@@ -167,6 +172,11 @@
 		reg = <0x0 0xfe410000 0x0 0x1000>;
 		#address-cells = <1>;
 		#size-cells = <1>;
+
+		pmu_io_domains: io-domains {
+			compatible = "rockchip,rk1808-pmu-io-voltage-domain";
+			status = "disabled";
+		};
 	};
 
 	qos_npu: qos@fe850000 {
-- 
2.35.3

